migrator/migrator.go

```diff
@@ -157,7 +162,23 @@ func (p *PouchMigrator) PreMigrate(ctx context.Context, takeOverContainer bool)
 		return nil
 	}
 
-	containersDir := path.Join(p.pouchHomeDir, "containers")
+	// Get all volumes on host.
+	volumes, err := p.dockerd.VolumeList()
+	if err != nil {
+		return fmt.Errorf("failed to get volumes list: %v", err)
+	}
+	// check volume's driver, now only support local driver
+	for _, vol := range volumes {
+		if vol.Driver != "local" {
+			return fmt.Errorf("d2p only support local volume driver, now contains %s driver", vol.Driver)
```

Comment:
s/now/not ? 
Actually I cannot quite follow you.

Reply:
Actually, what I mean is that we only support local driver now, but this host contains other volumes that created by other drivers.

Maybe my expression is to confusion 😆 i will updated it